docs/usage/notifications.md

```diff
@@ -27,4 +27,28 @@ Note: The notification of an entity deletion can either provides from the real d
 ## Digest
 A digest allows triggering the sending of notifications based on `multiple triggers` over a given period.
 
+## Notifiers
 
+### Connectors
+
+OpenCTI as some built-in notifier connectors that can be used as notifier in for Notification and Activity alerting. 
+Connectors are registered with a schema describing how the connector will interact.
+For example, the webhook connector has the following schema:
+- A verb (GET, POST, PUT, ...)
```

Comment:
This is not display as a list
![image](https://github.com/OpenCTI-Platform/docs/assets/53513584/896edf42-fe15-4405-af6b-9292b6affbc8)